docs/ossm/multi-control-planes/README.md

```diff
@@ -0,0 +1,176 @@
+# Multiple Istio Control Planes in a Single Cluster
+By default, the control plane will read all configuration in all namespaces - see [Cluster wide by default](../create-mesh/README.md#cluster-wide-by-default) for details. To achieve [soft multi-tenancy](../create-mesh/README.md#soft-multi-tenancy), Istio provides [discoverySelectors](../create-mesh/README.md#discoveryselectors)  which together with revisions capability allows to install multiple control planes in a Single Cluster.
```

Comment:
```suggestion
By default, the control plane will read all configuration in all namespaces - see [Cluster wide by default](../create-mesh/README.md#cluster-wide-by-default) for details. To achieve [soft multi-tenancy](../create-mesh/README.md#soft-multi-tenancy), Istio provides [discoverySelectors](../create-mesh/README.md#discoveryselectors)  which together with revisions enables you to install multiple control planes in a single cluster.
```

Reply:
Ah, those links are broken now after updates to create-mesh doc. I will fix it.